test/spaces/visitSpace.test.js

```diff
@@ -335,12 +336,14 @@ const hasPhaseLayout = async (
             break;
           }
           default:
+            // eslint-disable-next-line no-console
```

Comment:
I think these per-line disables for no-console are getting a bit out of control. Should we remove that rule at the `.eslintrc` level?

Reply:
I think it is nice to keep this rule (for instance when I debug I let some console.log by accident 😅 )

Reply:
One way to address this for now is to always use `console.debug` for debugging. And not allow that. Or only allow `console.error`. Or something like that. Is it at all possible with ESLint?